scripts/compile.sh

```diff
@@ -96,4 +102,18 @@ elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
         -DUSE_MFEM=${USE_MFEM}
   make -j8
 fi
+
 popd
+
+if [[ $INSTALL_PYTHON == "true" ]]; then
+    . ${REPO_PREFIX}/scripts/python_setup.sh
+    cd ${REPO_PREFIX}/lib
+    ${REPO_PREFIX}/dependencies/swig/swig_install/bin/swig -I${REPO_PREFIX}/dependencies/swig/swig_install/share/swig/4.0.2 -I${REPO_PREFIX}/dependencies/swig/swig_install/share/swig/4.0.2/python -I${REPO_PREFIX}/dependencies/swig/swig_install/share/swig/4.0.2/std -c++ -python -py3 librom.i
+    mv librom_wrap.cxx $LIB_BUILD_DIR
+    mv librom.py $LIB_BUILD_DIR
+    cd $LIB_BUILD_DIR
+
+    # TODO: Untie the Python lib from LC.
+    mpicxx -O2 -fPIC -DSWIG -c librom_wrap.cxx -I/usr/include/python3.6m -I/usr/workspace/huynh24/libROM4/env3/lib/python3.7/site-packages/mpi4py/include -I${REPO_PREFIX}/lib
+    mpicxx -shared -L/usr/workspace/huynh24/libROM4/env3/lib/python3.7/site-packages -Wl,-rpath,$LIB_BUILD_DIR -L$LIB_BUILD_DIR librom_wrap.o -lmkl_scalapack_lp64 -lmkl_blacs_intelmpi_lp64 -liomp5 -lpthread -lm -ldl -lROM -o _librom.so
```

Comment:
This path must be updated. It is tied to my username.

Reply:
This line converts the libROM wrap .o file into a .so Python library that is importable.